src/components/common/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -43,20 +43,23 @@
 
             <!-- Mailing Address -->
             <td class="px-0">
-              <base-address class="director-detail" :address="orgPerson.mailingAddress" />
+              <MailingAddress class="director-detail" :address="orgPerson.mailingAddress" />
```

Comment:
The kebab-case component name doesn't allow you to F12 on it to go to its definition. The CamelCase name does (in most cases).